message/options.go

```diff
@@ -107,10 +107,27 @@ func (options Options) path(buf []byte) (int, error) {
 // Return's number of used buf bytes or error when occurs.
 func (options Options) Path() (string, error) {
 	buf := make([]byte, 32)
-	m, err := options.path(buf)
+	m, err := options.path(buf, URIPath)
+	if err == ErrTooSmall {
+		buf = append(buf, make([]byte, m)...)
+		m, err = options.path(buf, URIPath)
+	}
+	if err != nil {
+		return "", err
+	}
+	buf = buf[:m]
+	return string(buf), nil
+}
+
+// LocationPath joins Location-Path options by '/' to the buf.
+//
+// Return's number of used buf bytes or error when occurs.
+func (options Options) LocationPath() (string, error) {
```

Comment:
Don't we need a setter or why do only getter is added? 

If I'm reading [rfc7252](https://www.rfc-editor.org/rfc/rfc7252) correctly, LocationPath should be contained in a POST response. If somebody uses go-coap to construct the response we should provide a convenient way to append the option to the message.

Reply:
Yes. Thx

Reply:
Sorry for asking again: Yes = squash, or yes = keep them separate?

Reply:
Squash it, Jozef wants to squash everything all the time so I'm 100% certain that's what he meant. :D